docs_src/design/adr/devops/0007-Release-Automation.md

```diff
@@ -2,7 +2,9 @@
 
 ## Status
 
-Approved by TSC 04/08/2020
+Original Approved by TSC 04/08/2020
+
+Updates In Review for Hanoi
```

Comment:
So if we approve this PR, will you update this line before we merge it?

Reply:
I removed any date changes from my PR to avoid any circular references, I'm not sure how we handle this scenario. 

I can add a line back to reference a TSC approval date after the fact if that's the norm.